docs/programming-oak.md

```diff
@@ -488,8 +488,8 @@ to the room:
 [embedmd]:# (../examples/chat/module/rust/src/lib.rs Rust /.*self\.rooms\.entry\(/ /\}\);$/)
 ```Rust
                 let channel = self.rooms.entry(label.clone()).or_insert_with(|| {
-                    let (wh, rh) =
-                        oak::io::channel_create(&label).expect("could not create channel");
+                    let (wh, rh) = oak::io::channel_create("Room initial channel", &label)
```

Comment:
I suspect we can just omit "channel" from any channel name, it should be pretty obvious that they are channels from the context in which they are printed / displayed (hopefully).

Reply:
Removed "channel" from the names everywhere.